core/src/free/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="pref_cast_message">@string/pref_cast_message_free_flavor</string>
+</resources>
```

Comment:
At the end of file, right? That's something I really never cared about. Is it ever relevant other than a style choice?